### PR TITLE
add rule 018 tests

### PR DESCRIPTION
**Pull request by theolextego**

### CODE REVIEW

File: Rule018Tests.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
...ps://frmfaas.sybrin.com/function/off-typology-processor\",\n                \"cfg\": \"030@1.0\",\n                \"rules\": [\n                  {\n                    \"id\": \"003@1.0\",\n                    \"host\": \"http://openfaas:8080\",\n                    \"cfg\": \"1.0\"\n                  },\n                  {\n                    \"id\": \"006@1.0\",\n                    \"host\": \"http://openfaas:8080\",\n                    \"cfg\": \"1.0\"\n                  }\n                ]\n              },\n              {\n                \"id\": \"031@1.0\",\n                \"host\": \"https://frmfaas.sybrin.com/function/off-typology-processor\",\n                \"cfg\": \"031@1.0\",\n                \"rules\": [\n                  {\n                    \"id\": \"003@1.0\",\n                    \"host\": \"http://openfaas:8080\",\n                    \"cfg\": \"1.0\"\n                  },\n                  {\n                    \"id\": \"007@1.0\",\n                    \"host\": \"http://openfaas:8080\",\n                    \"cfg\": \"1.0\"\n                  }\n                ]\n              }\n            ]\n          }\n        ]\n      }\n    ]\n  }\n}\n",
```
Comment (johanfol) [inline Rule018Tests.postman_collection.json:39]: Please use internal names instead of external

**Review by johanfol — CHANGES_REQUESTED** [review]

**Review by uche-lextego — APPROVED** [review]
LGTM

File: Rule018Tests.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+							"    pm.expect(jsonData.ruleResult.result).to.be.eql(true);",
+							"});",
+							"",
+							"pm.test(\"should have rule result with insufficient history reason\", function () {",
+							"    pm.expect(jsonData.ruleResult.reason).to.be.eql('Exceptionally large outgoing transfer detected');",
+							"});",
+							"",
+							"pm.test(\"Remove Seed Data from DB\", function () {",
+							"    pm.sendRequest({",
+							"        url: 'https://frmarango.sybrin.com/_db/transactionHistory/_api/cursor',",
```
Comment (jonty-esterhuizen) [inline Rule018Tests.postman_collection.json:821]: this needs to internal urls

Comment (jonty-esterhuizen) [reply Rule018Tests.postman_collection.json:821]: http://arango.development:8529/

**Review by jonty-esterhuizen — CHANGES_REQUESTED** [review]

**Review by jonty-esterhuizen — APPROVED** [review]

**Review by johanfol — APPROVED** [review]

**Pull request merged**